pictures/tasks.py

```diff
@@ -41,6 +44,21 @@ def _process_picture(
         picture = utils.reconstruct(*picture)
         picture.delete()
 
+    if field:
+        app_label, model_name, _ = field.split(".")
+        sender = apps.get_model(app_label=app_label, model_name=model_name)
+    else:
+        sender = _process_picture
+
+    signals.process_picture_done.send(
+        sender=sender,
+        storage=storage.deconstruct(),
```

Comment:
Now we're inside the process, and you can pass pointers to instances.
```suggestion
        storage=storage,
```

Reply:
Of course!